deployments/ohw-hub/config/staging.yaml

```diff
@@ -9,13 +9,13 @@ jupyterhub:
       hosts:
         - staging.ocean.hackweek.io
       letsencrypt:
-        contactEmail: scottyh@uw.edu
+        contactEmail: mayorga@apl.washington.edu
```

Comment:
This should be something for HTTPS: Scott's email is the one that the hackweek.io domain is registered under. I'll revert this line.